packages/orbit-components/config/fetchIcons.mts

```diff
@@ -85,7 +83,8 @@ try {
   }
 }
 
-const api = <T extends unknown>(url: string) =>
+// @ts-expect-error MTS doesn't support generics yet for that
```

Comment:
For that what? 🤔 and why don't we define the type if we know it? It will ensure types are defined on the function calls and returned values, no?